src/LogViewer/index.js

```diff
@@ -106,6 +114,17 @@ export default class LogViewer extends React.Component {
   handleMessage(e) {
     if (e.data && typeof e.data === 'object') {
       this.setState({ data: e.data });
+
+      Object.keys(e.data).forEach((name) => {
+        switch(name) {
+          case 'highlightError':
+            return this.highlightError(e.data.highlightError);
+          case 'lineNumber':
+            return this.jumpToLine(e.data.lineNumber);
+          case 'highlightStart':
+            return this.setState({ highlightStart: e.data.highlightStart, highlightEnd: e.data.highlightEnd });
```

Comment:
The state is acting weird when the logviewer is embedded inside an iframe. `this.setState` updates the state but the change is not seen unless I call setState again (send another postMessage). 
Would you happen to know why this is happening? The state seems to be updating which is why I am a little bit confused as to why it's not highlighting on the first call.